ui-core/src/styles/inputs/statusMessage.css

```diff
@@ -31,3 +31,60 @@
     }
   }
 }
+
+.status-message-wrapper-tooltip {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  width: 320px;
+  top: 74px;
+  gap: 12px;
+  border-radius: 8px;
+  box-shadow:
+    0 10px 20px 0 rgba(0, 0, 0, 0.2),
+    0 1px 0 rgba(255, 255, 255, 0.6) inset;
+  padding: 6px 12px 10px;
+  min-height: 64px;
+  z-index: 10;
+
+  &.small {
+    top: 58px;
+  }
```

Comment:
Sorry to insist, but if `top` was calculated with `top: calc(100% - 16px - 3px - 4px);`, you shouldn't need to add this `.small` style here and the `small` props to `StatusMessage`. (16px and 3px are `padding-bottom` from `.feed-pack` and `.input-wrapper`).
We could even add in main.css. to improve maintainability :
```
:root {
  --input-wrapper-padding: 3px;
  --field-wrapper-padding-bottom: 16px;
  --status-message-wrapper-tooltip--offset: 4px;
}
```
and use them in for `top` in `status-message.css` (gonna ask for opinions in #ui channel)